src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEFieldSymbol.cs

```diff
@@ -413,7 +413,8 @@ public override Accessibility DeclaredAccessibility
                         break;
 
                     default:
-                        throw ExceptionUtilities.UnexpectedValue(_flags & FieldAttributes.FieldAccessMask);
+                        access = Accessibility.Private;
```

Comment:
There are only eight values possible

---
In reply to: [64623823](https://github.com/dotnet/roslyn/pull/11547#discussion_r64623823) [](ancestors = 64623823)